src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -361,9 +360,7 @@ export const SearchBarDropdown = ({
     searchHistory,
   ])
 
-  const showBNBComingSoonBadge = Boolean(
-    chainId !== undefined && chainId === SupportedChainId.BNB && !isLoading && !CHAIN_ID_TO_BACKEND_NAME[chainId]
-  )
+  const showBNBComingSoonBadge = Boolean(chainId !== undefined && chainId === SupportedChainId.BNB && !isLoading)
```

Comment:
nit: do you need the chainId !== undefined? i think chainId === SupportedChainId.BNB is enough